starboard/android/shared/drm_system.cc

```diff
@@ -206,10 +212,34 @@ DrmSystem::DrmSystem(
     return;
   }
   j_media_crypto_ = env->ConvertLocalRefToGlobalRef(j_media_crypto_);
+
+  Start();
+}
+
+void DrmSystem::Run() {
+  JniEnvExt* env = JniEnvExt::Get();
+  bool result = env->CallBooleanMethodOrAbort(
+      j_media_drm_bridge_, "createMediaCryptoSession", "()Z");
+  if (result) {
+    created_media_crypto_session_.store(true);
+  }
+  if (!result && j_media_crypto_) {
+    env->DeleteGlobalRef(j_media_crypto_);
+    j_media_crypto_ = NULL;
+  }
+
+  ScopedLock scoped_lock(mutex_);
+  if (!deferred_session_update_requests_.empty()) {
```

Comment:
nit: the check is redundant.  Feel free to keep it as is as I don't have a strong preference here.